default/AI/FreeOrionAI.py

```diff
@@ -75,27 +64,30 @@ def startNewGame(aggression=fo.aggression.aggressive):  # pylint: disable=invali
         return
 
     turn_timer.start("Server Processing")
-    print "New game started, AI Aggression level %d (%s)" % (aggression, UserString(_aggression_names[aggression]))
 
     # initialize AIstate
     global foAIstate
-    foAIstate = AIstate.AIstate(aggression=aggression)
+    foAIstate = AIstate.AIstate(aggression=_aggression)
+    aggression = foAIstate.character.get_behavior(Aggression)
+    print "New game started, AI Aggression level %d (%s)" % (
+        aggression.key, aggression_name[foAIstate.character])
     foAIstate.session_start_cleanup()
     print "Initialized foAIstate class"
     planet_id = PlanetUtilsAI.get_capital()
     universe = fo.getUniverse()
     if planet_id is not None and planet_id != -1:
         planet = universe.getPlanet(planet_id)
-        new_name = " ".join([random.choice(_capitals.get(aggression, []) or [" "]).strip(), planet.name])
-        print "Capitol City Names are: ", _capitals
+        possible_capitals = aggression_capitals[foAIstate.character]
```

Comment:
If the character defines the capitals, why not let it be handled by the character object?```character.get_possible_capitals()```